src/main/docs/guide/dbc/dbcRepositories.adoc

```diff
@@ -34,4 +34,20 @@ As you can see from the above example, using abstract classes can be useful as i
 
 The example above uses the api:data.jdbc.runtime.JdbcOperations[] interface which simplifies executing JDBC queries within the context of transactions.
 
-TIP: You could also inject whichever other tool you wish to use to handle more complex  queries, such as QueryDSL, JOOQ, Spring JdbcTemplate etc.
+====
+
```

Comment:
if the property `datasources.default.transactionManager` = 'springJdbc' is required, add it to the documentation.